src/rules/index.ts

```diff
@@ -0,0 +1,378 @@
+} from './padding-around-describe-blocks'
+import {
+  default as paddingAroundExpectGroups,
+  RULE_NAME as paddingAroundExpectGroupsName,
+} from './padding-around-expect-groups'
+import {
+  default as paddingAroundTestBlocks,
+  RULE_NAME as paddingAroundTestBlocksName,
+} from './padding-around-test-blocks'
+import {
+  default as preferCalledExactlyOnceWith,
+  RULE_NAME as preferCalledExactlyOnceWithName,
+} from './prefer-called-exactly-once-with'
+import {
+  default as preferCalledOnce,
+  RULE_NAME as preferCalledOnceName,
+} from './prefer-called-once'
+import {
+  default as preferCalledTimes,
+  RULE_NAME as preferCalledTimesName,
+} from './prefer-called-times'
+import {
+  default as preferCalledWith,
+  RULE_NAME as preferCalledWithName,
+} from './prefer-called-with'
+import {
+  default as preferComparisonMatcher,
+  RULE_NAME as preferComparisonMatcherName,
+} from './prefer-comparison-matcher'
+import {
+  default as preferDescribeFunctionTitle,
+  RULE_NAME as preferDescribeFunctionTitleName,
+} from './prefer-describe-function-title'
+import {
+  default as preferEach,
+  RULE_NAME as preferEachName,
+} from './prefer-each'
+import {
+  default as preferEqualityMatcher,
+  RULE_NAME as preferEqualityMatcherName,
+} from './prefer-equality-matcher'
+import {
+  default as preferExpectAssertions,
+  RULE_NAME as preferExpectAssertionsName,
+} from './prefer-expect-assertions'
+import {
+  default as preferExpectResolves,
+  RULE_NAME as preferExpectResolvesName,
+} from './prefer-expect-resolves'
+import {
+  default as preferExpectTypeOf,
+  RULE_NAME as preferExpectTypeOfName,
+} from './prefer-expect-type-of'
+import {
+  default as preferHooksInOrder,
+  RULE_NAME as preferHooksInOrderName,
+} from './prefer-hooks-in-order'
+import {
+  default as preferHooksOnTop,
+  RULE_NAME as preferHooksOnTopName,
+} from './prefer-hooks-on-top'
+import {
+  default as preferImportInMock,
+  RULE_NAME as preferImportInMockName,
+} from './prefer-import-in-mock'
+import {
+  default as preferImportingVitestGlobals,
+  RULE_NAME as preferImportingVitestGlobalsName,
+} from './prefer-importing-vitest-globals'
+import {
+  default as preferLowercaseTitle,
+  RULE_NAME as preferLowercaseTitleName,
+} from './prefer-lowercase-title'
+import {
+  default as preferMockPromiseShorthand,
+  RULE_NAME as preferMockPromiseShorthandName,
+} from './prefer-mock-promise-shorthand'
+import {
+  default as preferSnapshotHint,
+  RULE_NAME as preferSnapshotHintName,
+} from './prefer-snapshot-hint'
+import {
+  default as preferSpyOn,
+  RULE_NAME as preferSpyOnName,
+} from './prefer-spy-on'
+import {
+  default as preferStrictBooleanMatchers,
+  RULE_NAME as preferStrictBooleanMatchersName,
+} from './prefer-strict-boolean-matchers'
+import {
+  default as preferStrictEqual,
+  RULE_NAME as preferStrictEqualName,
+} from './prefer-strict-equal'
+import {
+  default as preferToBeFalsy,
+  RULE_NAME as preferToBeFalsyName,
```

Comment:
we can also get rid of this repetitive RULE_NAME constant since all rules's names matches with their filename basename. We actually don't need it it.

Reply:
Fair point! Just pushed ca4e99c. Does this look better to you?